README.md

```diff
@@ -19,10 +19,7 @@
 
 <img src="/docs/images/format/CarbonData_logo.png" width="200" height="40">
 
-Apache CarbonData(incubating) is a new big data file format for faster
-interactive query using advanced columnar storage, index, compression
-and encoding techniques to improve computing efficiency, in turn it will 
-help speedup queries an order of magnitude faster over PetaBytes of data. 
+Apache CarbonData(incubating) is a fully indexed columnar and Hadoop native data format for processing heavy analytical workloads and detailed queries on big data.
```

Comment:
I suggest following to describe CarbonData in one setence. 
Apache CarbonData(incubating) is a fully indexed columnar data format for fast analytics on big data platform, e.g. Apache Hadoop, Apache Spark, etc.

Reason: by nature, CarbonData files are indexed and columnar, and the goal is make all analytics faster. 
@gvramana @ravipesala @JihongMA @Vimal-Das @jarray888 what is your opinion?